### PR TITLE
swagger2markup-cli 1.3.3 (new formula)

### PR DESCRIPTION
**Pull request by thirtified**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/swagger2markup-cli.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  bottle :unneeded
+  depends_on :java => "1.8+"
+
+  def install
+    libexec.install "swagger2markup-cli-#{version}.jar"
+    bin.write_jar_script libexec/"swagger2markup-cli-#{version}.jar", "swagger2markup"
+  end
+
+  test do
+    system bin/"swagger2markup"
```
Comment (fxcoudert) [inline Formula/swagger2markup-cli.rb]: Can we run the tool on a simple example? Check that its output is sane?

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/swagger2markup-cli.rb (hunk 1 above)
Comment (thirtified) [reply Formula/swagger2markup-cli.rb]: Thanks for the suggestion, I have added a basic test.

File: Formula/swagger2markup-cli.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        /test:
+          get:
+            responses:
+              "200":
+                description: Describe the test resource
+    EOS
+    # convert swagger spec to asciidoc
+    system bin/"swagger2markup convert -i test.yaml -f test"
+    # sanity check: first line should be title line with specified name
+    `head -n 1 test.adoc`.strip == "= TestSpec"
```
Comment (fxcoudert) [inline Formula/swagger2markup-cli.rb]: Use `shell_output` and `assert_equal`

File: Formula/swagger2markup-cli.rb [hunk 3]
```diff
@@ -0,0 +1,40 @@
+class Swagger2markupCli < Formula
+  # Original description from GitHub (too long for HomeBrew):
```
Comment (fxcoudert) [inline Formula/swagger2markup-cli.rb]: Remove comments

File: Formula/swagger2markup-cli.rb (hunk 1 above)
Comment (fxcoudert) [reply Formula/swagger2markup-cli.rb]: Great!

File: Formula/swagger2markup-cli.rb [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        description: Example Swagger spec
+      host: localhost:3000
+      paths:
+        /test:
+          get:
+            responses:
+              "200":
+                description: Describe the test resource
+    EOS
+    # convert swagger spec to asciidoc
```
Comment (fxcoudert) [inline Formula/swagger2markup-cli.rb]: Remove comments

**Review by fxcoudert — CHANGES_REQUESTED** [review]

**Comment by thirtified** [comment]
Thanks @fxcoudert for the help – suggestions added.

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @thirtified for your pull request!

**Pull request merged**